py3status/modules/moc.py

```diff
@@ -16,9 +16,9 @@
         (default '\?if=is_started [\?if=is_stopped \[\] moc|
         [\?if=is_paused \|\|][\?if=is_playing >] {title}]')
     sleep_timeout: sleep interval for this module. when moc is not running,
-        this interval will be used. this allows some flexible timing where one
-        might want to refresh constantly with some placeholders... or to refresh
-        only once every minute rather than every few seconds. (default 20)
+        this interval will be used. this allows one to refresh constantly
+        with time placeholders and/or to refresh once every minute rather
+        than every few seconds (default 20)
```

Comment:
This still reads a bit clucky to me.  I think it is not an easy thing to get across